src/box/tuple_format.c

```diff
@@ -789,6 +813,39 @@ tuple_format_new(struct tuple_format_vtab *vtab, void *engine,
 	return NULL;
 }
 
+/**
+ * Check that @a constr1_count constraints @a constr1 are tolerant to any
+ * thing that satisfy @a constr2_count constraints @a constr2.
+ */
+static bool
+constraints_are_tolerant(const struct tuple_constraint *constr1,
```

Comment:
Awkward name IMO. Better call `tuple_constraints1_can_store_constraints2_tuples` for consistency with `tuple_format1_can_store_format2_tuples`.